src/cloudant/replicator.py

```diff
@@ -60,27 +60,46 @@ def create_replication(self, source_db=None, target_db=None,
 
```

Comment:
The existing `user_ctx` applies to a target only. I think we need to make `source_user(_ctx?)` and `target_user(_ctx?)` args that can be used to specify. We should alias the `user_ctx` to `target_user` to maintain backwards compatibility.

Reply:
Not sure I understand what you mean here. Can we please go through it when you get a mo? Thanks.

Reply:
It was a long time ago and I'm struggling to reverse engineer my meaning too! I think what I was getting at was that the `user_ctx` default setting is dependent on some checks on the `target`, but not the source. My suggestion above doesn't make sense either though.

What I think makes sense is that the default `user_ctx` is set if one isn't supplied, but should not depend on any checks for the source or target database.

Reply:
Done. Removed admin party check in eaaa7f1.